update_docker.sh

```diff
@@ -17,15 +17,11 @@ fi
 
 docker run -u $(id -u) --rm -v "$PWD:/app" "$IMAGE_NAME"
 
-# UPDATE_FILE is an indicator we can use in Jenkins
-# to run other jobs if there was, in fact, an update
-rm -f "$UPDATE_FILE"
-
 if [[ "$1" == "commit" ]]; then
     if git status --porcelain | grep -E "\.json$"; then
         git add ./releases/
         git commit -m "Update release data"
-        git rev-parse HEAD > "$UPDATE_FILE"
+        git push
```

Comment:
to which repo will this push? I think with this setup that would be Gitlab? We'll probably need to do something like `git push git@github.com/mozilla/release-notes.git`